pkg/apis/k0sctl.k0sproject.io/v1beta1/cluster/host.go

```diff
@@ -589,6 +589,9 @@ func (h *Host) NeedCurl() bool {
 }
 
 // NeedIPTables returns true when the iptables package is needed on the host
+//
+// Deprecated: iptables is only required for k0s versions that are unsupported
+// for a long time already (< v1.22.1+k0s.0).
```

Comment:
I think k0sctl still works for something like k0s v0.10.0 before the scheme switch.

The upgrade smoke currently starts from v1.21.6+k0s.0.

Should we start removing other stuff that was added to make k0sctl work with versions that are now unsupported? cc: @jnummelin